src/util.ml

```diff
@@ -1224,7 +1228,7 @@ value string_of_fevent_name conf base efam_name =
 
 value string_of_witness_kind conf p witness_kind =
   match witness_kind with
-  [ Witness -> transl_nth conf "witness/witness/witnesses" 0
+  [ Witness -> transl_nth conf "witness/witnesses" 0
```

Comment:
You should remove `witness/witnesses` key and use `witness/witness/witnesses` everywhere.